src/templates/TabPage.tsx

```diff
@@ -1,106 +1,118 @@
 import { graphql } from 'gatsby'
-import get from 'lodash/get'
 import React from 'react'
 import { Helmet } from 'react-helmet'
 import NavTabs from '../components/NavTabs'
-import Layout from '../layouts/Default'
+import Section from '../components/Section'
+import DefaultLayout from '../layouts/Default'
 
 class TabPageTemplate extends React.Component {
   render() {
-    const site = get(
-      this.props,
-      'data.contentfulPageTab.page___main[0].site[0]',
-    )
-    const page = get(this.props, 'data.contentfulPageTab.page___main[0]')
-    const tabs = get(this.props, 'data.contentfulPageTab.page___main[0].tabs')
-    const tab = get(this.props, 'data.contentfulPageTab')
+    const { site, pages, pageLookup, pageStructure } = this.props.pageContext
+    const pageDetails = this.props.data.contentfulSitePage
+    const page = {
+      ...pageStructure,
+      ...pageDetails,
+    }
+    if (page.sections === null) {
+      page.sections = []
+    }
 
     return (
-      <Layout>
-        <div>
-          <Helmet title={`${tab.title} // ${site.title}`} />
-          <h1>{tab.title}</h1>
-
-          <NavTabs page={page} tabs={tabs} />
-
-          <div>
-            <p>{tab.updatedAt}</p>
-
-            {tab.sections.map((section) => {
-              return (
-                <div>
-                  <h2>{section.title}</h2>
+      <DefaultLayout
+        site={site}
+        pages={pages}
+        pageLookup={pageLookup}
+        page={page}
+      >
+        <Helmet title={`${page.title} < ${site.title}`} />
```

Comment:
Suggestion (non-blocking): this line could be moved to the `DefaultLayout` component since it's repeated in your two pages. We could even pass a prop to `DefaultLayout` if we need to override the title, like:

```jsx
<DefaultLayout title="My custom title">...</DefaultLayout>
```